interface/src/avatar/AvatarManager.cpp

```diff
@@ -767,6 +767,7 @@ RayToAvatarIntersectionResult AvatarManager::findRayIntersectionVector(const Pic
         glm::vec3 rayDirectionInv = { rayDirection.x != 0.0f ? 1.0f / rayDirection.x : INFINITY,
                                       rayDirection.y != 0.0f ? 1.0f / rayDirection.y : INFINITY,
                                       rayDirection.z != 0.0f ? 1.0f / rayDirection.z : INFINITY };
+        glm::vec3 viewFrustumPos = BillboardModeHelpers::getPrimaryViewFrustumPosition();
```

Comment:
getPrimaryViewFrustumPosition isn't super expensive, but any slight improvement to picking is helpful.  here (and elsewhere below) we get the value once at the beginning, and then pass it down so we don't have to call it multiple times